packages/web/src/pages/modals/Modals.tsx

```diff
@@ -46,7 +46,7 @@ import { UploadConfirmationModal } from 'components/upload-confirmation-modal'
 import { USDCPurchaseDetailsModal } from 'components/usdc-purchase-details-modal/USDCPurchaseDetailsModal'
 import { USDCTransactionDetailsModal } from 'components/usdc-transaction-details-modal/USDCTransactionDetailsModal'
 import TierExplainerModal from 'components/user-badges/TierExplainerModal'
-import ConnectedUserListModal from 'components/user-list-modal/ConnectedUserListModal'
```

Comment:
do we get to remove ConnectedUserListModal entirely, or not yet?

Reply:
oh good point, need to be more strategic here